python/tchannel/zipkin/formatters.py

```diff
@@ -97,6 +98,8 @@ def json_formatter(traces, *json_args, **json_kwargs):
 
 
 def ipv4_to_int(ipv4):
+    if ipv4 == 'localhost':
+        ipv4 = '127.0.0.1'
```

Comment:
inet_aton is a system api to convert api to a binary form. http://linux.die.net/man/3/inet_aton. It doesn't support 'localhost'